docker/InternVL_Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG CUDA_VERSION=cu12
+
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04 AS cu12
```

Comment:
Please use `openmmlab/lmdeploy:latest-cu11` and `openmmlab/lmdeploy:latest-cu12`, respectively

Reply:
> Please use `openmmlab/lmdeploy:latest-cu11` and `openmmlab/lmdeploy:latest-cu12`, respectively, so that we don't need to build lmdeploy and its dependencies any more.

i have change the base image in InternVL_Dockerfile